### PR TITLE
Aux base alert fix

### LINKED ISSUE #66254: "<span class='danger'>Launch sequence activated! Prepare for drop!!</span>"
## Round ID:

local server running on the latest tgcode

## Testmerges:

none

## Reproduction:

image

### PR DESCRIPTION
**Pull request by TheBoondock**
## About The Pull Request
The warning for the aux base warning alert uses say() instead of audible_message and thus it doesn't work with span_danger, this pr changes it to audible_message.
Fixes #66254

## Why It's Good For The Game
Bug fixes good

## Changelog

:cl:
fix: fixes aux base console outputting "span_danger()" in launch sequence
/:cl:

### CODE REVIEW

**Comment by TheBonded** [comment]
wait fuck ignore that im just stupid

File: code/modules/mining/aux_base.dm [hunk 1]
```diff
@@ -144,7 +144,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/computer/auxiliary_base, 32)
 				return
 			var/shuttle_error = SSshuttle.moveShuttle(shuttleId, params["shuttle_id"], 1)
 			if(launch_warning)
-				say(span_danger("Launch sequence activated! Prepare for drop!!"))
+				audible_message(span_danger("Launch sequence activated! Prepare for drop!!"))
```
Comment (MrMelbert) [inline code/modules/mining/aux_base.dm:147]: Keep it as a say, and just remove the span from around the message.

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/modules/mining/aux_base.dm (hunk 1 above)
Comment (TheBoondock) [reply code/modules/mining/aux_base.dm:147]: I want to preserve the span class of the message since its warning and all

Comment (MrMelbert) [reply code/modules/mining/aux_base.dm:147]: Then pass the danger span into the say.

`say(>>insert message here<<, spans = list(danger))

File: code/modules/mining/aux_base.dm [hunk 2]
```diff
@@ -144,7 +144,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/computer/auxiliary_base, 32)
 				return
 			var/shuttle_error = SSshuttle.moveShuttle(shuttleId, params["shuttle_id"], 1)
 			if(launch_warning)
-				say(span_danger("Launch sequence activated! Prepare for drop!!"))
+				say("Launch sequence activated! Prepare for drop!!", span = list("danger"))
```
Comment (MrMelbert) [inline code/modules/mining/aux_base.dm:147]:
```suggestion
				say("Launch sequence activated! Prepare for drop!!", spans = list("danger"))
```

**Pull request merged**